templates/job_working_dir.job.j2

```diff
@@ -1,8 +1,7 @@
-Universe = vanilla
+Universe = local
```

Comment:
Does this mean the job is actually running on the node where it is submitted? So on the maintainece node?

Reply:
That is interesting and I agree that makes a lot of sene ... the question is just why do we submit them via condor at all and not just execute them? So we need a scheduler for those jobs? I.e. can there be so many jobs that they need to be queued?

Reply:
Yes, this now runs on the maintenance node

```bash
galaxy@maintenance:/tmp$ /data/dnb01/maintenance/htcondor_crontab_scheduling_submitter.sh
Submitting job(s).
1 job(s) submitted to cluster 1.
Submitting job(s).
1 job(s) submitted to cluster 2.
Submitting job(s).
1 job(s) submitted to cluster 3.

galaxy@maintenance:/tmp$ condor_q

-- Schedd: maintenance.galaxyproject.eu : <10.5.68.158:9618?... @ 07/27/23 16:22:05
OWNER  BATCH_NAME    SUBMITTED   DONE   RUN    IDLE  TOTAL JOB_IDS
galaxy ID: 1        7/27 16:22      _      1      _      1 1.0
galaxy ID: 2        7/27 16:22      _      _      1      1 2.0
galaxy ID: 3        7/27 16:22      _      _      1      1 3.0

Total for query: 3 jobs; 0 completed, 0 removed, 2 idle, 1 running, 0 held, 0 suspended
Total for galaxy: 3 jobs; 0 completed, 0 removed, 2 idle, 1 running, 0 held, 0 suspended
Total for all users: 3 jobs; 0 completed, 0 removed, 2 idle, 1 running, 0 held, 0 suspended
```

Reply:
At first, I thought of simply running it as an individual cron job while migrating as there were only 3 job files (don't remember the details of the involved shell scripts) and then saw that logs are collected on `data/dnb01/maintenance/`. Since I do not know the historic reasons for this, I updated the `universe` so the jobs can still be submitted as a condor job and the logs it was collecting can go without interruption.

Reply:
Ok, valid reason. We should look at this at some point. And maybe scheduling these jobs is eiherway is a good idea.